app/_src/networking/transparent-proxying.md

```diff
@@ -0,0 +1,190 @@
+---
+title: Transparent Proxying
+---
+
+## What is Transparent Proxying?
+A transparent proxy is a type of server that can intercept network traffic to and from a service without changes to the client application code. In the case of {{site.mesh_product_name}} it is used to capture traffic and redirect it to `kuma-dp` so Mesh policies can be applied.
+
+To accomplish this, {{site.mesh_product_name}} utilizes [`iptables`](https://linux.die.net/man/8/iptables) and offers additional, experimental support for [`eBPF`](https://ebpf.io/). The examples provided in this section will concentrate on iptables to clearly illustrate the point.
```

Comment:
Might want to link to https://deploy-preview-1661--kuma.netlify.app/docs/2.6.x/production/dp-config/cni/#merbridge-cni-with-ebpf instead of ebpf.io